web/server/vue-cli/src/components/Statistics/CheckerCoverage/CheckerCoverageStatistics.vue

```diff
@@ -0,0 +1,334 @@
+import {
+  ReviewStatusMixin,
+  SeverityMixin,
+  ToCSV
+} from "@/mixins";
+import { AnalysisInfoHandlingAPIMixin } from "@/mixins/api";
+import { BaseStatistics } from "@/components/Statistics";
+import CheckerCoverageStatisticsTable from "./CheckerCoverageStatisticsTable";
+import CheckerCoverageStatisticsDialog from "./CheckerCoverageStatisticsDialog";
+import {
+  MAX_QUERY_SIZE,
+  ReportFilter,
+  RunFilter
+} from "@cc/report-server-types";
+import { ccService, handleThriftError } from "@cc-api";
+import {
+  CheckerInfoAvailability
+} from "@/mixins/api/analysis-info-handling.mixin";
+import TooltipHelpIcon from "@/components/TooltipHelpIcon";
+
+export default {
+  name: "CheckerCoverageStatistics",
+  components: {
+    CheckerCoverageStatisticsTable,
+    CheckerCoverageStatisticsDialog,
+    TooltipHelpIcon
+  },
+  mixins: [
+    AnalysisInfoHandlingAPIMixin,
+    BaseStatistics,
+    ReviewStatusMixin,
+    SeverityMixin,
+    ToCSV
+  ],
+
+  data() {
+    return {
+      checker_stat: {},
+      loading: false,
+      noProperRun: false,
+      problematicRuns: [],
+      runs: null,
+      runData: [],
+      selectedCheckerName: null,
+      showRuns: {
+        enabled: false,
+        disabled: false,
+        problematic: false
+      },
+      statistics: [],
+      type: null,
+    };
+  },
+
+  computed: {
+    actualRunNames() {
+      return this.runs.filter(run => !this.problematicRuns.map(
+        problematicRun => problematicRun.runId
+      ).includes(run.runId)).map(run => run.runName);
+    }
+  },
+
+  watch: {
+    checker_stat(stat) {
+      this.statistics = Object.keys(stat).map(checker_id => {
+        return {
+          checker: stat[checker_id].checkerName,
+          severity: stat[checker_id].severity,
+          enabledInAllRuns: stat[checker_id].disabled.length === 0
+            ? 1
+            : 0,
+          enabledRunLength: stat[checker_id].enabled.length,
+          disabledRunLength: stat[checker_id].disabled.length,
+          closed: stat[checker_id].closed.toNumber(),
+          outstanding: stat[checker_id].outstanding.toNumber(),
+        };
+      });
+    },
+
+    async runIds() {
+      this.noProperRun = false;
+    },
+
+    "$route"() {
+      if (this.runs && this.$route.query.run !== this.actualRunNames) {
+        this.$router.go();
+      }
+    }
+  },
+
+  methods: {
+    downloadCSV() {
+      const data = [
+        [
+          "Checker Name", "Severity", "Status",
+          "Closed Reports", "Outstanding Reports",
+        ],
+        ...this.statistics.map(stat => {
+          return [
+            stat.checker,
+            this.severityFromCodeToString(stat.severity),
+            stat.enabledInAllRuns
+              ? "Enabled in all selected runs"
+              : "Not enabled in all selected runs",
+            stat.closed,
+            stat.outstanding
+          ];
+        })
+      ];
+
+      this.toCSV(data, "codechecker_analysis_statistics.csv");
```

Comment:
(The downloaded filename are not in line with the name of the component it is getting obtained from.)